src/components/Contacts/ContactListTableRow.jsx

```diff
@@ -14,48 +14,42 @@ import { DocumentListContext } from '@contexts';
 import { StyledTableCell, StyledTableRow } from '../Table/TableStyles';
 
 /**
- * @typedef {import("../../typedefs.js").clientListTableRowProps} clientListTableRowProps
+ * @typedef {import("../../typedefs.js").contactListTableRowProps} contactListTableRowProps
  */
 
 /**
- * ClientListTableRow Component - Component that generates the individual table
- * rows of clients from data within ClientList
+ * ContactListTableRow Component - Component that generates the individual table
+ * rows of contacts from data within ContactList
  *
- * @memberof Clients
- * @name ClientListTableRow
- * @param {clientListTableRowProps} Props - Props for ClientListTableRow
- * @returns {React.JSX.Element} The ClientListTableRow Component
+ * @memberof Contacts
+ * @name ContactListTableRow
+ * @param {contactListTableRowProps} Props - Props for ContactListTableRow
+ * @returns {React.JSX.Element} The ContactListTableRow Component
  */
-const ClientListTableRow = ({ client, setShowDeleteClientModal, setSelectedClientToDelete }) => {
+const ContactListTableRow = ({ contact, deleteContact }) => {
   const theme = useTheme();
   const [pinned, setPinned] = useState(false);
-  const { setClient } = useContext(DocumentListContext);
+  const { setContact } = useContext(DocumentListContext);
 
   // determine what icon gets rendered in the pinned column
   const pinnedIcon = pinned ? <PushPinIcon color="secondary" /> : <PushPinOutlinedIcon />;
 
-  // Event handler for pinning client to top of table
+  // Event handler for pinning contact to top of table
   // ***** TODO: Add in moving pinned row to top of table
   const handlePinClick = () => {
     setPinned(!pinned);
   };
 
-  // Event handler for deleting a client from client list
-  const handleSelectClientToDelete = () => {
-    setSelectedClientToDelete(client);
-    setShowDeleteClientModal(true);
-  };
-
   return (
     <StyledTableRow>
       <StyledTableCell align="center">
         <Link
-          to={`/profile/${encodeURIComponent(client.webId)}`}
-          state={{ client }}
+          to={`/profile/${encodeURIComponent(contact.webId)}`}
+          state={{ contact }}
           style={{ textDecoration: 'none', color: theme.palette.primary.dark }}
         >
-          <Button sx={{ textTransform: 'capitalize' }} onClick={() => setClient(client)}>
-            {client.person}
+          <Button sx={{ textTransform: 'capitalize' }} onClick={() => setContact(contact)}>
+            {`${contact.person}`}
```

Comment:
Minor nitpick (it's fine as it is), but think we could straight-up use the `String` function for this given we're just trying to force a string from this value.